### PR TITLE
feature/leaderboard refresh

### PR DESCRIPTION
**Pull request by haideralishah**
leaderboard Refresh Screens UIs and functionality.

### CODE REVIEW

**Comment by sagararyal** [comment]
- [ ] Use the Profile Picture

- [ ] Font issue
Screen Shot 2020-01-08 at 1 29 13 PM

File: app/components/LeaderboardRefresh/header.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        alignItems: 'center',
+        marginHorizontal: 15,
+        backgroundColor: 'white'
+      }}
+    >
+      <View>
+        <Image
+          source={{
+            uri:
+              'https://cdn.zeplin.io/5cad97d9b33d0b5931d90a9b/assets/C4D2C402-B10F-4E0D-BB89-588E7669E98F.png'
```
Comment (norbertschuler) [inline app/components/LeaderboardRefresh/header.js]: Shouldn't we host the images ourself?

File: app/components/LeaderboardRefresh/Countries/CountriesLeaderBoard.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                style={styles.oneContryContainer}
+              >
+                <View style={styles.indexContainer}>
+                  <Text style={styles.indexText}>{index + 1}</Text>
+                </View>
+                <View style={styles.countryFlagContainer}>
+                  <Image
+                    style={styles.countryFlagImage}
+                    source={{
+                      uri: `https://www.trilliontreecampaign.org/flags/png/256/${countryCode}.png`
```
Comment (norbertschuler) [inline app/components/LeaderboardRefresh/Countries/CountriesLeaderBoard.js]: As in the near future we maybe switch the domain of the website, we should only use relative path names for links to our own server. Does the backend provide these flags, so they are not part of our application?

**Comment by norbertschuler** [comment]
As in other PRs I reviewed today the header is broken on iOS13.3 with iPhone X simulator (but that better should be fixed in another branch and then merged into develop and all other PRs):

Simulator Screen Shot - iPhone 11 - 2020-01-10 at 15 33 51

**Comment by norbertschuler** [comment]
I really can not recommend merging one feature branch into another one. All my experience in all the years says this only leads into serious trouble, e.g. merge conflicts et al. - just my opinion as software architect.

File: app/components/LeaderboardRefresh/Countries/CountriesLeaderBoard.js (hunk 2 above)
Comment (sagararyal) [reply app/components/LeaderboardRefresh/Countries/CountriesLeaderBoard.js]: @haideralishah we should not hardcode URL, get site url instead as defined in index.js

**Comment by sagararyal** [comment]
@norbertschuler reverted the merge of project api. branch and merged develop into this.

**Pull request merged**